orc8r/cloud/go/service/middleware/unary/identity_decorator.go

```diff
@@ -118,7 +118,7 @@ func SetIdentityFromContext(ctx context.Context, _ interface{}, info *grpc.Unary
 		if len(snlist) != 1 {
 			// there can be only one CSN, error out if not
 			log.Printf("Multiple CSNs found in metadata: %+v", ctxMetadata)
-			err = status.Error(codes.Unauthenticated, "Invalid SCN List")
+			err = status.Error(codes.Unauthenticated, "Multiple CSNs present")
```

Comment:
When the call (from directoryd) makes it to the state service, it calls `protos.GetClientGateway(ctx)`. Won't this currently fail since we we're replacing the client CSN with the orc8r value?